scania-digital-design-system/tegel
# docs(github-template): removing request help template

**Describe pull-request**  
Removing the request help template 

**Solving issue**  
Removing the request help template as it is not used

**How to test**  
1. Check if file is deleted
2. Real check will happen after merging, then under issues, there should be no "Request Help" template, only "Bug" and "Feature request" options.
